composer/core/engine.py

```diff
@@ -321,12 +321,15 @@ def _compile(
         Returns:
             Sequence[Algorithm]: Modified sequence of algorithms.
         """
-        from composer.algorithms import CutMix, MixUp, SelectiveBackprop, StochasticDepth
+        from composer.algorithms import CutMix, MixUp, SelectiveBackprop, StochasticDepth, FusedLayerNorm
 
         # Move selective backprop to the beginning while maintaining order of other algorithms
         algorithms = sorted(algorithms_to_run,
                             key=lambda x: not isinstance(x, SelectiveBackprop) and not isinstance(x, StochasticDepth))
 
+        # Move fused layernorm to the end while maintaining order of other algorithms (FLN only does surgery on leaf modules)
+        algorithms = sorted(algorithms_to_run, key=lambda x: isinstance(x, FusedLayerNorm))
```

Comment:
I presume we want to keep the previous sort from line 327?

```suggestion
        algorithms = sorted(algorithms, key=lambda x: isinstance(x, FusedLayerNorm))
```

Reply:
Yes, thank you!